### PR TITLE
Feature RTMD-88 - refactor of report page

### PR DESCRIPTION
**Pull request by joefitter**
* removed cucmber acceptance tests
* added codecept
* updated report page fields and content
* removed report page multi report functionality
* removed confirm controller and tests
* updated report tests
* added acceptance tests for report page
* clean up scripts and deps

### CODE REVIEW

**Comment by JoeChapman** [comment]
building is failing with `/bin/sh: 1: /test/test.sh: Permission denied` becoz you need `chmod +x test.sh` and do `chmod +x run.sh` and possibly even `chmod +x .drone.yml` while you're at it. Just to be sure

File: .drone.yml [hunk 1]
```diff
@@ -5,19 +5,32 @@ pipeline:
...
+    when:
+      event: pull_request
 
   build_app:
     image: docker:1.11
     environment:
       - DOCKER_HOST=tcp://127.0.0.1:2375
     commands:
-      - docker build -t app .
+      - docker build -t rotm .
```
Comment (JoeChapman) [inline .drone.yml]: reason why you changed this? BTW, if you change it here you should change value you tag the image has with https://github.com/UKHomeOffice/rotm/blob/3538874d3180eba77076388a4e7920f8b95143dc/.drone.yml#L93

Comment (joefitter) [inline .drone.yml]: ah balls - I missed one

Comment (joefitter) [inline .drone.yml]: yeah, updated it to be consistent with other apps

File: Dockerfile-acceptance [hunk 2]
```diff
@@ -1,7 +1,18 @@
...
 
-COPY acceptance_tests /code
-WORKDIR /code
-RUN bundle install --deployment
+WORKDIR /test
 
-ENTRYPOINT "cucumber"
+ENV PHANTOM_JS phantomjs-2.1.1-linux-x86_64
+
+RUN wget https://github.com/Medium/phantomjs/releases/download/v2.1.1/${PHANTOM_JS}.tar.bz2 && \
```
Comment (easternbloc) [inline Dockerfile-acceptance]: i still need to get that running as another container don't like that we have to do this

Comment (JoeChapman) [inline Dockerfile-acceptance]: another day

**Pull request merged**